build-system/tasks/extension-generator/index.js

```diff
@@ -330,6 +333,9 @@ async function makeExtension() {
   const bundleConfig = await (argv.bento
     ? makeBentoExtension()
     : makeAmpExtension());
+  if (!bundleConfig) {
+    return;
+  }
```

Comment:
This looks like it's introducing a silent failure. I think it should be accompanied by a warning.
```suggestion
  if (!bundleConfig) {
    log(yellow('WARNING:'), 'Could not write extension files.');
    return;
  }
```